### PR TITLE
Simple whitespace cleanup

### PR DESCRIPTION
**Pull request by tekknolagi**
This makes the spacing in the file more consistent

### CODE REVIEW

File: src/refmt_impl.ml [hunk 1]
```diff
@@ -209,8 +209,9 @@ let () =
           raise (Invalid_config ("Invalid --parse setting for interface '" ^ s ^ "'."))
         )
       in
-      let _ = if parsedAsInterface then
-          raise (Invalid_config ("The file parsed does not appear to be an implementation file.")) in
+      let _ =
+          if parsedAsInterface then
+              raise (Invalid_config ("The file parsed does not appear to be an implementation file.")) in
```
Comment (yunxing) [inline src/refmt_impl.ml]: Maybe move 'in' to the next line so it is more consistent? 

(This is why we need reason to automatically make the file consistent!)

**Comment by chenglou** [comment]
Made the change myself in the online editor. Github magic.

**Comment by chenglou** [comment]
Aaaaaand I forgot a spot. Forget about that magic.

Pushed a new change.

**Comment by tekknolagi** [comment]
Oh alright! I will go ahead and change the other instance.

**Pull request merged**